gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureJavaLocalCodegenPlugin.java

```diff
@@ -159,7 +159,8 @@ private static void createGenerateTask(
                                 sanitizePackageName(project.getGroup().toString()));
                         return properties;
                     }));
-                    task.getOutputDirectory().set(project.file(ConjurePlugin.JAVA_GENERATED_SOURCE_DIRNAME));
+                    task.getOutputDirectory()
+                            .set(project.getLayout().getProjectDirectory().dir("src/generated/java"));
```

Comment:
I completely forgot this output directory from #1401, I should have moved this over to the new build directory